src/tiny_sqlite.nim

```diff
@@ -134,6 +135,10 @@ proc skipLeadingWhiteSpaceAndComments(sql: var cstring) =
         else:
             return
 
+proc resetStmt(stmtHandle: sqlite.Stmt) =
+    discard sqlite.reset(stmt_handle)
```

Comment:
nitpicking: why `stmt_handle` and not `stmtHandle`?

Reply:
The curse of style insensitivity... It's been fixed now